README.md

```diff
@@ -9,24 +9,18 @@ as well as other repositories that are important for maintenance.
 
 It uses [vue-cli](https://github.com/vuejs/vue-cli/blob/dev/docs/cli.md)
 to simplify local development and production builds for VueJS.
-You can use it on https://github-issue-search-client-silverstripe.vercel.app.
+You can use it on <https://silverstripe.github.io/github-issue-search-client/>.
```

Comment:
This URL will change again once we have a solid domain - but it's better than having the vercel URL here in the meantime.